example/example_c/template.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
```

Comment:
since the example is so short, I think it should belong to a test case (e.g. using a test case with such input, but do not put it in the example folder)